src/website/views/templates/portal.pug

```diff
@@ -882,12 +1042,158 @@ h3 Alert
+            button.chi-button.-icon.-close(aria-label='Close')
+              .chi-button__content
+                i.chi-icon.icon-x
+          .chi-drawer__content
+            .-p--2.-text Drawer content here
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-portal-drawer
+      li.-disabled
+        a(href=`#webcomponent-portal-drawer`) Web Component
+      li.-active
+        a(href=`#html-portal-drawer`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-portal-drawer
+  .chi-tabs-panel.-active#html-portal-drawer
+    .chi-tab__description
+      span.-text--grey
+        | This HTML Blueprint requires JavaScript.
+        | You may use your own solution, or use Chi's vanilla JavaScript solution,
+        | <a href="../../getting-started/installation">Chi.js</a>.
+    :code(lang="html")
+      <!-- Trigger -->
+      <button id="drawer-trigger" class="chi-button -primary chi-drawer__trigger" data-target="#drawer">Launch demo drawer</button>
+
+      <!-- Drawer -->
+      <div class="chi-backdrop -closed">
+        <div class="chi-backdrop__wrapper">
+          <div id="drawer" class="chi-drawer -bottom -portal">
+            <div class="chi-drawer__header">
+              <span class="chi-drawer__title">
+                Drawer title here
+              </span>
+              <button class="chi-button -icon -close" aria-label="Close">
+                <div class="chi-button__content">
+                  <i class="chi-icon icon-x"></i>
+                </div>
+              </button>
+            </div>
+            <div class="chi-drawer__content">
+              <div class="-p--2 -text">Drawer content here</div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- JavaScript -->
+      <script>chi.drawer(document.getElementById('drawer-trigger'));</script>
+
+h2 Accordion
+.example.-mb--3
+  .-p--3
+    .chi-accordion.-portal#example-portal
+      .chi-accordion__item
+        button.chi-accordion__trigger
+          .chi-accordion__title
+            | Accordion 1
+          i.chi-icon.icon-chevron-down
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+      .chi-accordion__item.-expanded
+        button.chi-accordion__trigger
+          .chi-accordion__title
+            | Accordion 2
+          i.chi-icon.icon-chevron-down
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+      .chi-accordion__item
+        button.chi-accordion__trigger
+          .chi-accordion__title
+            | Accordion 3
+          i.chi-icon.icon-chevron-down
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+      .chi-accordion__item
+        button.chi-accordion__trigger
+          .chi-accordion__title
+           | Accordion 4
+          i.chi-icon.icon-chevron-down
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-portal-accordion
```

Comment:
Tab hrefs do not match the chi-tabs-panel id's (e.g. html-portal-accordion vs html-accordion-portal)

Reply:
@mattnickles good catch!